sample_playbook/ipam/address_block/delete_address_block.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- hosts: localhost
+  connection: local
+  collections:
+    - infoblox.b1ddi_modules
+
+  vars:
+  - host: "{{ Host server to connect to }}"
+  - api: "{{ api key to access the server }}"
+
+  tasks:
+    - name: Delete Address Block
+      b1_ipam_address_block:
+        address: "40.0.0.0/24"
+        space: "qa-space-1"
+        api_key: "{{ api }}"
+        host: "{{ host }}"
+        state: absent
+
+    - name: Delete Address Block

```

Comment:
Multiple delete task example is redundant here. I feel one task should be sufficient.

Reply:
Addressed.